packages/eth-rpc-adapter/package.json

```diff
@@ -7,10 +7,10 @@
   "scripts": {
     "clean": "rm -rf lib tsconfig.tsbuildinfo",
     "build": "tsc --build tsconfig.json",
-    "test:e2e": "SKIP_PUBLIC=true mocha -r tsconfig-paths/register src/__tests__/e2e/*",
+    "test:e2e": "SKIP_PUBLIC=true vitest --run --config vitest.config.e2e.ts",
     "start": "ts-node -r tsconfig-paths/register src/index.ts",
-    "dev": "ts-node-dev -T -r tsconfig-paths/register src/index.ts | pino-pretty --singleLine --colorize --ignore time,hostname,jsonrpc",
-    "test:CI": "mocha -r tsconfig-paths/register src/__tests__/e2e/* --timeout 300000",
+    "dev": "ts-node-dev -T -r tsconfig-paths/register src/index.ts -l | pino-pretty --singleLine --colorize --ignore time,hostname,jsonrpc",
+    "test:CI": "vitest --run --config vitest.config.e2e.ts",
```

Comment:
currently we have two vitest configs, I am wondering if there is any issue if we just use one config for all tests? We usually don't have to need to run { normal, e2e } tests seperately, and in CI we should run all tests

Reply:
regarding e2e I can't find a better away to exclude e2e to sharedConfig and include them on config.e2e